cs/v1/CS_v1.0.0.yaml

```diff
@@ -663,104 +739,61 @@ components:
         - carrierExportVoyageNumber
         - timestamps
       properties:
-        carrierServiceName:
-          type: string
-          x-stoplight:
-            id: g8e6j84n6mcw8
-          description: The name of the service.
-          maxLength: 50
-          example: Great Lion Service
-        carrierServiceCode:
```

Comment:
Are all `carrier`-specific properties (`carrierServiceCode`, `carrierServiceName`, `carrierExportVoyageNumber` etc...) being removed and only the 'universal'-ones are kept? This seems wrong to me?